bridgesync/processor.go

```diff
@@ -388,6 +391,17 @@ type SetClaim struct {
 	CreatedAt   uint64      `meddler:"created_at"`
 }
 
+// BackwardLET representation of a BackwardLET event,
+// that is emitted by the bridge contract when a claim is backwarded.
+type BackwardLET struct {
+	BlockNum             uint64      `meddler:"block_num"`
+	BlockPos             uint64      `meddler:"block_pos"`
+	PreviousDepositCount *big.Int    `meddler:"previous_deposit_count,bigint"`
+	PreviousRoot         common.Hash `meddler:"previous_root,hash"`
+	NewDepositCount      *big.Int    `meddler:"new_deposit_count,bigint"`
+	NewRoot              common.Hash `meddler:"new_root,hash"`
+}
```

Comment:
Maybe it would make sense to keep the `BackwardLET` indexing in a separate PR, until we have a full clarity and we implement the `ForwardLET` feature. WDYT?